spec/statesman/adapters/active_record_spec.rb

```diff
@@ -376,6 +376,16 @@
         expect(adapter.last.to_state).to eq("y")
       end
     end
+
+    context "without previous transitions" do
+      it "does query the database only once" do
+        expect(model.my_active_record_model_transitions).
+          to receive(:order).once.and_call_original
+
+        expect(adapter.last).to eq(nil)
+        expect(adapter.last).to eq(nil)
+      end
+    end
```

Comment:
This test verifies that the caching of the `#last` method works even when there are no transitions been made yet.